static/css/_card.scss

```diff
@@ -0,0 +1,5 @@
+@import 'variables';
+
+md-card{
+  outline:0;
+}
```

Comment:
not necessary to have its own scss file, originally it has one dedicated scss in angular-material